scripts/src/main/resources/scripts/command/graalvm

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# $1: optional setup
+function doSetup() {
+  installDir="${DEVON_IDE_HOME}/software/extra/graalvm"
+  if doIsMacOs
+  then
+    GRAALVM_HOME="${installDir}/Contents/Home"
+  else
+    GRAALVM_HOME="${installDir}"
+  fi
+
+  doDevonCommand java setup silent
+  if [ "${1}" != "silent" ] || [ ! -d "${installDir}" ]
+  then
+    local version="${GRAALVM_VERSION:-22.0.0.2}"
+    local javaVersion="${EXTRA_JAVA_VERSION}"
+    if [ -z "${javaVersion}" ]
+    then
+      javaVersion="${JAVA_VERSION}"
+    fi
+    if [ -n "${javaVersion}" ]
+    then
+      doVersionCompare "${javaVersion}" "11u0"
+      if [ "${?}" == 2 ]
+      then
+        # Java version is lower than 11
+        doConfirmWarning "GraalVM requires Java 11 or newer.\nHowever, [EXTRA_]JAVA_VERSION is ${javaVersion} - in this setup GraalVM can not work.\nPlease update JAVA_VERSION or set EXTRA_JAVA_VERSION in your settings/devon.properties!"
+      fi
+    fi
+    doInstall "-" "${installDir}" "graalvm" "${version}"
+
+    graalvm_export="export GRAALVM_HOME=${GRAALVM_HOME}"
+    if ! grep -q "${graalvm_export}" "${DEVON_IDE_HOME}/conf/devon.properties"
+    then
+      echo -e "\n${graalvm_export}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
+    fi
+  fi
+
+  if [ "${1}" != "silent" ] && ! doIsQuiet
+  then
+    local java_binary="${GRAALVM_HOME}/bin/java"
+    if [ ! -f "${java_binary}" ]
+    then
+      java_binary="${GRAALVM_HOME}/bin/java.exe"
+    fi
+    doRunCommand "'${java_binary}' -version" "verify installation of GraalVM"
+  fi
+}
+
+# Call GraalVM with specified arguments
+function doRun() {
+  doSetup silent
+  command="$GRAALVM_HOME/bin/$*"
+  local binary=${command%% *}
```

Comment:
here we are removing a potential asterisk?
Maybe I am not getting what is really going on here...